### DISCUSSION TITLE
Group 3 Discussion - Design decisions

### DISCUSSION THREAD

**Discussion by patrickpatrickpatrick** [alphagov/govuk-design-system#4334] [Community events]
Discussion for Group 3 of the Design System Hack Day - Linking Design Systems

**Comment by leighmaude** [comment]
Hello!

**Reply by onupama1** [reply]
hello

**Comment by OlagokeD** [comment]
Hi

**Comment by Cha-gandhi** [comment]
Hello!

**Comment by AllyBranley** [comment]
Hello!

**Comment by BNArch** [comment]
Hello!

**Comment by AllyBranley** [comment]
I've copied the task from the Padlet, for us to refer to:

1) Understand why and how design systems made the design decision.

2) As a department design system, how do we support teams across our department to make good decisions about what components to use that (a) create consistency for users across our departments products and services, and (b) support our department plans to improve maintenance and standards.

3) More case studies
Publish case studies on the results of using a particular component, or pattern. A deep dive on the rational on why a component was chosen, backup by user research
I like this! This can help in understanding why the decision has been made in that way and also useful when presenting to stakeholders

**Comment by sallytolson1** [comment]
Hello everyone online, we're in a room now, shall we join a call?

**Comment by BNArch** [comment]
![Design System _hack_ Day 2024-11-28 01_52 PM_Page 1- 1732807502527](https://github.com/user-attachments/assets/ac77c26c-1112-4e82-bd2c-3b87a0a4d786)

**Comment by ImranH-GDS** [comment]
Hello, we have about 30 minutes left. Can each group/ subgroup think about who will play back your ideas during the show and tell?

**Comment by BNArch** [comment]


**Reply by BNArch** [reply]
![Uploading Design System _hack_ Day 2024-11-28 01_52 PM_Page 1-[1732809755992].png…]()

**Comment by BNArch** [comment]
![Design System _hack_ Day 2024-11-28 01_52 PM_Page 1- 1732809755992](https://github.com/user-attachments/assets/3a28ceab-0864-4dc4-930d-a385946a92fe)
